RFC-002-mapeo-protocol.md

```diff
@@ -206,6 +206,14 @@ compatible version.
 An implementor should only use the major (`N.x.x`) version number. A minor
 (`x.N.x`) and patch (`x.x.N`) version are included for possible future use.
 
+## 3.4 `Ping()`
+
+A peer should sent periodic `Ping` messages in order to ensure that the
+connection is still active. TCP does not do this automatically, and it can take
+a long time for the kernel to detect that the connection has been lost.
+
+Q: how long between PINGs? how many missed to timeout connection?
```

Comment:
Good idea.

Also I'd be interested in discussing ACKs here, to find which data the other side has (via bitfield) perhaps?